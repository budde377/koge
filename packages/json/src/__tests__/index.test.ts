import { json, out } from ".."


describe("json", () => { 
  it("Will generate object", async () => {
    const template = json`
        ${
  {
    foo: 1,
    bar: false
  }}
      `
    const output = out('path.json', template)
    expect(await output.data()).toMatchSnapshot()
  })
})

