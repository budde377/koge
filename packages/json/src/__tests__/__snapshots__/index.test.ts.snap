// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`json Will generate object 1`] = `
"{
  \\"foo\\": 1,
  \\"bar\\": false
}"
`;
